server.js

```diff
@@ -21,9 +22,78 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({responseMessage: "Hi there! Type in /books to browse all books, or try /random-book to get a random book recommendation!"});
```

Comment:
You could add here something like this:
app.get('/', (req, res) => {
  res.json({
    responseMessage:
      'Hi there! Type in /books to browse all books, or try /random-book to get a random book recommendation!',
    data: listEndpoints(app),
  });
});
and import import listEndpoints from 'express-list-endpoints'; >> then you. will have your all endpoints listed on the "/" page